test/runtests.jl

```diff
@@ -87,16 +83,14 @@ end
         @time @safetestset "Symbol and integer based indexing of interpolated solutions" begin
             include("downstream/symbol_indexing.jl")
         end
-        if VERSION >= v"1.8"
-            @time @safetestset "Symbol and integer based indexing of integrators" begin
-                include("downstream/integrator_indexing.jl")
-            end
-            @time @safetestset "Problem Indexing" begin
-                include("downstream/problem_interface.jl")
-            end
-            @time @safetestset "Solution Indexing" begin
-                include("downstream/solution_interface.jl")
-            end
+        @time @safetestset "Symbol and integer based indexing of integrators" begin
```

Comment:
Someone needs to find the time to change all of these to 

```julia
@time @safetestset "Symbol and integer based indexing of integrators" include("downstream/integrator_indexing.jl")
```

This is what we agreed on as the right way, and the formatter is fine with it, but we just never got around to it. These kinds of clean up PRs would be a good time to just throw the regex on it.